assets/src/design-system/stories/index.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { dark, light } from '../theme/colors';
+
+export default {
+  title: 'DesignSystem/Colors',
+};
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+`;
+
+const Container = styled.div`
+  width: 200px;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const ColorBlock = styled.span`
+  width: 60%;
+  height: 60%;
+  border-radius: 50%;
+  border: 1px solid gray;
+  background-color: ${({ color }) => color};
+`;
+
+export const _default = () => {
+  const { brandColors, standard, accent, status } = dark;
+  const getColorSet = (color, idx) => (
+    <Container key={idx}>
+      <ColorBlock color={color} />
+      <p>{color}</p>
+    </Container>
+  );
```

Comment:
I think it could be nice to add some label here so under each color it shows a path to reference that color in the theme. 

Just would allow this to be a single point of reference

Reply:
great idea!